apidef/api_definitions.go

```diff
@@ -376,6 +376,8 @@ type APIDefinition struct {
 	EnableBatchRequestSupport bool                   `bson:"enable_batch_request_support" json:"enable_batch_request_support"`
 	EnableIpWhiteListing      bool                   `mapstructure:"enable_ip_whitelisting" bson:"enable_ip_whitelisting" json:"enable_ip_whitelisting"`
 	AllowedIPs                []string               `mapstructure:"allowed_ips" bson:"allowed_ips" json:"allowed_ips"`
+	EnableIpBlacklisting      bool                   `mapstructure:"enable_ip_blacklisting" bson:"enable_ip_blacklisting" json:"enable_ip_blacklisting"`
```

Comment:
Perhaps `EnableIPBlacklisting`?

Reply:
I was wondering the exact same thing and after checking, the word *blacklist* is both a noun and a verb. https://en.oxforddictionaries.com/definition/blacklist .

Reply:
I kept in same format as `EnableIpWhiteListing`